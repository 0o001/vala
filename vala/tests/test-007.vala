namespace Maman {
	class Bar {
		public virtual string do_action (int i) {
			return 1;
		}
	}
	
	class SubBar : Bar {
		public override string do_action (int i) {
			return 2;
		}
	
		public static int main (int argc, string[] argv) {
			Bar bar = new SubBar ();
		
			return bar.do_action (1);
		}
	}
}
