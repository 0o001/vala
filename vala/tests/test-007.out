Virtual Method Test: 1 2 3
